scripts/rpc-maas-tool.py

```diff
@@ -671,8 +671,9 @@ def _get_failed_checks(self, checks=None):
                 try:
                     result = self.rpcm.conn.test_existing_check(check)
                 except rackspace.RackspaceMonitoringValidationError as e:
-                    validation_error = (" Validation Error: %(s):"
-                                        % {'e': e.message})
+                    validation_error = " Validation Error: {s}:".format(
+                        s=e.message
+                    )
                     break
```

Comment:
I believe this is the final backport of three (plus the original master patch), so you might want to go back and look at fixing the patch where it has already landed in master/newton/liberty if this patch is broken